cpp/src/arrow/compute/kernels/scalar_string_test.cc

```diff
@@ -27,6 +27,7 @@
 #include <utf8proc.h>
 #endif
 
+#include "arrow/compute/api.h"
```

Comment:
nit: is there a more precise header we can include

Reply:
Removed.